app/src/main/java/org/jellyfin/androidtv/ui/preference/screen/PlaybackAdvancedPreferencesScreen.kt

```diff
@@ -64,6 +65,11 @@ class PlaybackAdvancedPreferencesScreen : OptionsFragment() {
 				setTitle(R.string.pref_external_player)
 				bind(userPreferences, UserPreferences.useExternalPlayer)
 			}
+
+			enum<ZoomMode> {
+				setTitle(R.string.lbl_zoom)
```

Comment:
We may want to add a new string resource that is more descriptive instead of this one

Reply:
Perhaps "Aspect Ratio" is more descriptive? I have made a new string resource called pref_aspect_ratio with this text.

Reply:
@nielsvanvelzen do you think "Aspect ratio" is descriptive enough? If not I can come up with another string that is more descriptive for this setting.

Reply:
Maybe something like "Default video zoom"?

Reply:
"Default video zoom" sounds good to me. One other suggestion could be "Default zoom mode" as the setting is under the Video category in the settings menu so not sure if the word "video" would be redundant. I think either could work but its hard for me to tell which one aligns better with the rest of settings.